src/Global/organisms/MainMenu.tsx

```diff
@@ -46,7 +46,38 @@ const SVGObjectText = styled.object`
   max-width: 180px;
 `;
 
-const NavigationContainer = styled.div``;
+const GradientContainer = styled.div`
+  &::after {
+    content: '';
+    background: radial-gradient(farthest-side at 50% 100%,rgba(246, 247, 249, 0) 1%, #F6F7F9 120%) center bottom; //Shodow bottom
+    height: 10px;
+    display: block;
+    position: absolute;
+    margin-top: -10px;
+    width: 300px;
```

Comment:
```suggestion
    width: 100%;
```

To fix overflow.

![image](https://user-images.githubusercontent.com/95204740/227992217-5d25d352-6b95-40bf-ad9a-0a02c0f2b440.png)